src/gh.js

```diff
@@ -31,48 +31,48 @@ async function getRegistrationToken() {
   }
 }
 
-async function removeRunner() {
-  const runner = await getRunner(config.input.label);
+async function removeRunners() {
+  const runners = await getRunners(config.input.label, true);
```

Comment:
what's this input boolean for? I don't see it in the function prototype

Reply:
Oh whoops that was used in an earlier version, will remove